openquake/server/views.py

```diff
@@ -766,6 +763,28 @@ def impact_get_rupture_data(request):
     return JsonResponse(rupdic, status=200)
 
 
+@csrf_exempt
+@cross_domain_ajax
+@require_http_methods(['POST'])
+def impact_get_stations_from_usgs(request):
+    """
+    Retrieve station data corresponding to a given usgs id
+
+    :param request:
+        a `django.http.HttpRequest` object containing usgs_id
+    """
+    user = request.user
+    user.testdir = None
```

Comment:
This should be the default, no need to specify it, right?

Reply:
We may add a property like we did for `User.level`